README.md

```diff
@@ -2,34 +2,30 @@
 <!-- markdownlint-disable first-line-h1 -->
 <div align="center">
   <a href="https://glaredb.com#gh-light-mode-only">
-    <img src="https://docs.glaredb.com/assets/logo.svg" height="44">
+    <img src="https://docs.glaredb.com/assets/logo.svg" height="60q">
   </a>
   <a href="https://glaredb.com#gh-dark-mode-only">
-    <img src="https://glaredb.com/logo.svg" height="44">
+    <img src="https://glaredb.com/logo.svg" height="60">
   </a>
 </div>
-
-<!-- Adds some spacing between logo and badges. -->
-<p></p>
+<br/>
+<p align="center">
+Data exists everywhere: on laptops, in databases like Postgres and Snowflake, and as files in S3, in various formats including Parquet, CSV, and JSON. Accessing and analyzing this data typically involves multiple steps across different systems.<br/> <br/> <b>GlareDB is designed to query your data wherever it lives using SQL that you
+already know.</b>
```

Comment:
I don't think I would be inclined to put this here, but if you are, can you use paragrpah tags and hard wrap it like the rest of the file, rather than doing the `br` thing.